### ISSUE TITLE
Elevated circulating guanidinoacetic acid concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7627]
**Preferred term label:**
Elevated circulating guanidinoacetic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increased concentration of guanidinoacetic acid in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Abnormal circulating proteinogenic amino acid derivative concentration HP:0033108

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 612736 
PMID: 15651030   4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
